csharp/src/Apache.Arrow/Apache.Arrow.csproj

```diff
@@ -14,7 +14,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>apache arrow</PackageTags>
     <Company>Apache</Company>
-    <Version>0.0.1</Version>
+    <Version>0.0.4</Version>
```

Comment:
? What's the reasoning for jumping from `1` to `4`?

Reply:
No reason. It was a mistake. We use a slightly patched version of Arrow internally and this change leaked in.